frontend/src/metabase-lib/v1/expressions/tokenizer/types.ts

```diff
@@ -1,20 +0,0 @@
-export enum OPERATOR {
```

Comment:
Moved to `metabase-lib/v1/expressions/punctuator` and renamed `PUNCTUATOR` (like [ECMA does](https://tc39.es/ecma262/#sec-punctuators)) to avoid confusion with `Lib.ExpressionParts.operator`.